daemon/api.go

```diff
@@ -1714,7 +1714,7 @@ func setSnapConf(c *Command, r *http.Request, user *auth.UserState) Response {
 	taskset, err := configstate.ConfigureInstalled(st, snapName, patchValues, 0)
 	if err != nil {
 		if _, ok := err.(*snap.NotInstalledError); ok {
-			return SnapNotFound(snapName, err)
+			return SnapNotFound(configstate.RemapSnapToResponse(snapName), err)
```

Comment:
we will not hit this case because core/system can be configured before anything is installed

Reply:
Removed now